_pages/detailed-process.md

```diff
@@ -59,7 +59,7 @@ This helps ensure consistency and fairness into the process. It also lets us col
 Before we start any selection we should:
 
 - Select interviewers (SMEs)
-    - We need 3 SMEs for resume review, and 4 for interviews. These can't be the same people for each candidate. So this points to the need for a fairly large pool. 8-10 is probably a *minimum* for most positions.
+    - We need 4 SMEs for interviews. These can't be the same people for each candidate. So this points to the need for a fairly large pool. 8-10 is probably a *minimum* for most positions.
```

Comment:
This is really minor, but if we're following AP style -- and I think that's handbook preference -- ten and below would be spelled out.